core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -857,7 +857,10 @@ public static void main(String[] args) {
           opcount = Integer.parseInt(props.getProperty(RECORD_COUNT_PROPERTY, DEFAULT_RECORD_COUNT));
         }
       }
-
+      if (threadcount > opcount){
```

Comment:
this looks unrelated to the README change. can you break it into a different PR please?

Reply:
Ok